srcs/juloo.keyboard2/Keyboard2.java

```diff
@@ -124,6 +124,45 @@ public void onCreate()
     _foldStateTracker.setChangedCallback(() -> { refresh_config(); });
   }
 
+  @Override
+  public boolean onKeyDown(int keyCode, KeyEvent event) {
+    InputConnection conn = getCurrentInputConnection();
+    if (!_config.move_cursor_with_volume || conn == null || !isInputViewShown()) {
```

Comment:
I think the `conn` check is not necessary. It's not a problem to skip it here as `_keyeventhandler` will do it again and it's not expected to be `null` in normal circumstances.